packages/vue3-vite/src/utils.ts

```diff
@@ -41,12 +41,29 @@ export function updateDependencies(
   devDeps: Record<string, string>
 ) {
   // Make sure we don't have dependency duplicates
+  ensureDepsInPackageJson(host);
```

Comment:
I add a new function here to make sure `dependencies` and `devDependencies` exists in `package.json` before adding/removing dependencies, so that we don't run into the same problem mentioned in #68.

Reply:
Got it!